targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/mbed_crc_api.c

```diff
@@ -0,0 +1,65 @@
+#include "crc_api.h"
+
+#include "drivers/fsl_crc.h"
+#include "platform/mbed_assert.h"
+
+#ifdef DEVICE_CRC
+
+static crc_bits_t width;
+
+bool hal_crc_is_supported(const crc_mbed_config_t* config)
+{
+  if (config == NULL)
+    return false;
```

Comment:
Please review coding style (hoewever targets/ are not checked), there are others that shall be fixed